src/argilla/training/spacy.py

```diff
@@ -35,18 +37,17 @@ def __init__(
         language: Optional[str] = None,
         gpu_id: Optional[int] = -1,
         model: Optional[str] = None,
+        optimize: Literal["efficiency", "accuracy"] = "efficiency",
```

Comment:
perhaps for me this makes sense to add to `update_config`too

Reply:
It's not so straight forward to modify that from the configuration file itself, so we should keep it as is, since we're using the `optimize` value to generate one config or the other initially, more things differ in those configuration files, not just a string as it may seem according to the `optimize` arg